### ISSUE TITLE
<code> tags appearing on public body admin page 

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#5924]
In the History section of a public body admin page /admin/bodies/xx 

`<code> `

tags are appearing as they are present in the following form in the HTML:

`&lt;code&gt;&lt;/code&gt;`

this isn't important in itself but it might indicate something more important has gone wrong.

**Comment by garethrees** [comment]
Did you have a specific example where you saw this?

I've just had a look through the timeline for bodies that you've recently edited to try to observe the problem, but I can't see it in any of those. Several seem to show `<code>` correctly (e.g. body 11219)

![Screenshot 2020-10-16 at 09 12 23](https://user-images.githubusercontent.com/282788/96232022-c7b1f280-0f8f-11eb-868f-e59b257a51a4.png)

**Comment by RichardTaylor** [comment]
The issue appears to occur on every body page where there's been email address,  home page, or publication scheme link added. 

I think the screenshot [posted](https://github.com/mysociety/alaveteli/issues/5924#issuecomment-709897825) shows the issue, surely the code tag should be a functional HTML tag affecting the way email addresses and links are displayed, the tag shouldn't be shown to the reader.

**Comment by mdeuk** [comment]
I've been able to reproduce this issue on body 4242.

Ordinarily, I'd expect the email address to appear with formatting, however we appear to be rendering the text without taking any notice of the code tag that is being inserted by Alaveteli, meaning we're displaying "<code></code>" rather than displaying rendered text.

<img src="https://user-images.githubusercontent.com/249418/96349564-603b9600-10a8-11eb-950b-efd293839fd3.png" width="550">

Viewing the raw HTML shows that we appear, as @RichardTaylor says, to be converting the tags so that they have <code>&lt;</code> and <code>&gt;</code> rendered as text rather than as machine readable code.

<img src="https://user-images.githubusercontent.com/249418/96349719-56666280-10a9-11eb-93d2-4a58ae79e037.png" width="550">

Could this be occurring because of the change to app/views/admin_public_body/show.html.erb in 52fd17ced?

**Comment by garethrees** [comment]
> I think the screenshot posted shows the issue…

🤦 right, sorry, I get you. I was expecting to see `&lt;code&gt;&lt;/code&gt;` and couldn't see it anywhere. 

> Could this be occurring because of the change to app/views/admin_public_body/show.html.erb in 52fd17c?

Yep, looks like it's that.

**Comment by MattK1234** [comment]
I've reopened this as the issue appears to still be occurring:

![image](https://user-images.githubusercontent.com/47503358/99915429-7ff35900-2cfb-11eb-9412-95a47ed375e0.png)


Noted on body 442. I've checked multiple bodies that @RichardTaylor updated today and it all seems to be the same.

They were all updated using the spreadsheet upload functionality in the admin interface.

Another example on body 232:

![image](https://user-images.githubusercontent.com/47503358/99915483-e37d8680-2cfb-11eb-9277-42301ae41be1.png)

Also on body 57680 which was not updated via the spreadsheet upload.
